stylize/clang_formatter.py

```diff
@@ -34,3 +35,14 @@ def run(self, args, filepath, check=False):
             proc.communicate()
             md5_after = file_md5(filepath)
             return md5_before != md5_after
+
+    def get_command(self):
+        if os.system("command -v clang-format >/dev/null") == 0:
```

Comment:
Yeah I was just looking around to see if there was anything nice for `compgen` and I don't see anything :/  I think the way you have it might be as good as it can be.